cmd/kyma/run/function/function.go

```diff
@@ -122,6 +124,7 @@ func (c *command) runContainer(ctx context.Context, client *client.Client, cfg w
 
 	if !c.opts.Detach {
 		fmt.Println("Logs from the container:")
+		fmt.Println("Container listen on the port: " + runtimes.ServerPort)
```

Comment:
What is the intention when printing: `Logs from the container:` and then in the next line `Container listen...`? To me the first line seems wrong here.

Also, second line should say: `Container listening on port: `.

Finally, usually instead of just using `fmt.Print`, check always if it would be better to use the `step` package the CLI has (not sure if needed in this case).

Reply:
The intention is to tell the user that the container is ready to use and the next logs should be sourced from the container, not from the `cli` itself

Reply:
Hi @pPrecel 
With the current 2 lines, I will see the following in my terminal:
```
Logs from the container:
Container listen on the port: XXXX
```

The first line is for sure wrong or missing something.